code/modules/xenoarcheaology/finds/find_spawning.dm

```diff
@@ -32,11 +32,11 @@
 	//icon_state
 	//item_state
 	switch(find_type)
-		if(1)
+		if(ARCHAEO_BOWL)
```

Comment:
how

Reply:
How what? The issues I constantly saw with the finds made the descriptions blank out half the time after being uncovered even with a field generator. Hadn't had that problem with current vorecode so this will be said fix. 👏